src/kava.js

```diff
@@ -478,18 +502,36 @@ class Kava extends BasePlugin {
   }
 
   _onFragLoaded(event: FakeEvent): void {
-    const seconds = Math.round(event.payload.miliSeconds) / 1000;
-    const fragResourceTimings = performance && performance.getEntriesByType('resource').filter(entry => entry.name == event.payload.url);
+    if (!this._fragLoadedFiredOnce) this._fragLoadedFiredOnce = true;
```

Comment:
eslint should yell about this - don't use one liner if's and always use curly brackets